.github/workflows/build-push-via-cicd-actions.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Push Image #by triggering cicd-actions workflow via Github API Call
+
+on:
+  push:
+    branches: [ feature/custatevec, feature/ZCLOUD-260/add-ci-with-github-actions ]
```

Comment:
we can probably keep feature/custatevec up and merge stuff into it, so this should work well with on merge into `feature/custatevec`